package/MDAnalysis/lib/formats/libmdaxdr.pyx

```diff
@@ -674,32 +701,40 @@ cdef class XTCFile(_XDRFile):
     """
     cdef float precision
 
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
     def _calc_natoms(self, fname):
         cdef int n_atoms
-        return_code = read_xtc_natoms(fname, &n_atoms)
+        cdef int return_code = read_xtc_natoms(fname, &n_atoms)
         return return_code, n_atoms
 
-
+    @cython.boundscheck(False)
```

Comment:
IIRC you can set this with a pragma like statement in a comment at the top of the file. We can probably always default to fast and dangerous.

Reply:
Yeah for sure can set with `# cython: boundscheck=False` but was just playing with the various functions. Will fix.